Jenkinsfile

```diff
@@ -2,4 +2,7 @@
  See the documentation for more options:
  https://github.com/jenkins-infra/pipeline-library/
 */
-buildPlugin(useContainerAgent: true)
+buildPlugin(configurations: [
+  [ platform: "docker", jdk: "8", jenkins: null ],
+  [ platform: "docker-windows", jdk: "8", jenkins: null ]
```

Comment:
Is there a windows build available for the Docker image you want to use? If not, we could just switch to a linux only config (e.g. https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/Jenkinsfile, they are using Docker too).

Reply:
> Is there a windows build available for the Docker image you want to use?
No, I just need MinIO which is based linux. FYI, I'm not sure about the windows agent but according to the documentation, this is the only windows agent with docker support (see https://github.com/jenkins-infra/documentation/blob/main/ci.adoc).

> If not, we could just switch to a linux only config (e.g. https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/Jenkinsfile, they are using Docker too).

I'm fine with linux only too.

Reply:
I changed it on the main branch directly.